OpenRA.Game/Traits/World/Country.cs

```diff
@@ -18,6 +18,9 @@ public class CountryInfo : TraitInfo<Country>
 		[Desc("This is the internal name for owner checks.")]
 		public readonly string Race = null;
 
+		[Desc("Pick a random race as the player's race out of this list.")]
+		public readonly string[] RandomRace = { };
```

Comment:
I'd be much happier if this were `RandomRaceMembers` or `RandomMembers`, atm this sounds like a bool.